src/persist/src/indexed/runtime.rs

```diff
@@ -383,10 +384,13 @@ impl RuntimeClient {
 
 /// A handle that allows writes of ((Key, Value), Time, Diff) updates into an
 /// [crate::indexed::Indexed] via a [RuntimeClient].
-#[derive(PartialEq, Eq)]
+// TODO: Are we OK with dropping Eq here? It seems to compile.
```

Comment:
yeah, we can always pull it back later if we need to (which doesn't even need to be left as a TODO imo)